src/classes/selectors/AppointmentResourcesSelector.cls

```diff
@@ -12,4 +12,25 @@ public inherited sharing class AppointmentResourcesSelector {
                 AND Appointment__r.Status = :CaseUtility.ASSIGNED_STATUS
         ];
     }
+
+    @AuraEnabled(Cacheable=true)
```

Comment:
MAJOR
Selector cannot be called from client-side. There must be a controller layer for that and controller calls selector.
Selector has 1 purpose - expose SOQL methods, so they can be reused by consumers.
More: https://www.apexhours.com/apex-enterprise-patterns/